zsl/constants.py

```diff
@@ -0,0 +1,8 @@
+class MimeType(object):
+    APPLICATION_JSON = 'application/json'
+    # type: str
+
+
+class HttpHeaders(object):
+    CONTENT_TYPE = 'Content-Type'
+    # type: str
```

Comment:
"It works on my pc" https://github.com/AtteqCom/zsl/blob/unit-test-refactoring/zsl/constants.py 

There is some problems with your commits as it always strips the new line in new files. Maybe it is CR/LF issue, maybe it's some quirky setting in your git config, which can strips these, I don't know, but it is still an issue.